CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
+- Check if record implement TemplateInstance to provide the support. See [quarkus-ls#974](https://github.com/redhat-developer/quarkus-ls/pull/974).
+- fix: separate user tag parameters inlay hint. See [quarkus-ls#968](https://github.com/redhat-developer/quarkus-ls/pull/968).
+- In qute, allow spaces between method parameters and only validate the return type of the method invocation when iterating over a method invocation. See [quarkus-ls#966](https://github.com/redhat-developer/quarkus-ls/issues/966).
+- fix: allow spaces inside section branch.. See [quarkus-ls#965](https://github.com/redhat-developer/quarkus-ls/pull/965).
+- fix: allow inject: with constructor class which have arguments. See [quarkus-ls#964](https://github.com/redhat-developer/quarkus-ls/pull/964).
+- Support for inner Enum in Qute files. See [quarkus-ls#963](https://github.com/redhat-developer/quarkus-ls/issues/963).
+- Support for void in property part of Qute template. See [quarkus-ls#961](https://github.com/redhat-developer/quarkus-ls/issues/961).
+- Template files generated by clicking on a code lens were missing the closing `</html>`. See [quarkus-ls#955](https://github.com/redhat-developer/quarkus-ls/issues/955).
+- quarkus.hibernate-orm.database.generation invalid validate value. See [quarkus-ls#945](https://github.com/redhat-developer/quarkus-ls/issues/945).
+- Qute fragments doesn't work with non inner Templates Java file. See [quarkus-ls#937](https://github.com/redhat-developer/quarkus-ls/issues/937).
+- inject:csrf is not retrieved in Qute template. See [quarkus-ls#901](https://github.com/redhat-developer/quarkus-ls/issues/901).
+- NPE with Workspace Symbol when LSP client return null as SymbolInformation List. See [lsp4mp#457](https://github.com/eclipse/lsp4mp/pull/457).
+- Unrecognized property mp.messaging.* when Channel annotation is used along with Multi. See [lsp4mp#451](https://github.com/eclipse/lsp4mp/pull/451).
+- fix: synchronized projectLabels + collect codeLens when project is loaded. See [lsp4mp#444](https://github.com/eclipse/lsp4mp/pull/444).
+
+### Build
+
+- Migrate Jenkins release job to GitHub Actions. See [#723](https://github.com/redhat-developer/vscode-quarkus/pull/723).
+
+### Other
+
+- Hide `Debug Quarkus` command from palette. See [#743](https://github.com/redhat-developer/vscode-quarkus/issues/743).
+- Use category `Quarkus` for all commands. See [#741](https://github.com/redhat-developer/vscode-quarkus/issues/741).
+- Bump gulp from 4.0.2 to 5.0.0. See [#713](https://github.com/redhat-developer/vscode-quarkus/pull/713).
+- Replace vulnerable pom-parser with fast-xml-parser in tests, and other updates. See [#657](https://github.com/redhat-developer/vscode-quarkus/pull/657).
+- Replace axios library with built-in fetch. See [#711](https://github.com/redhat-developer/vscode-quarkus/pull/711).
+- Fix tests for release workflow. See [vscode-microprofile#185](https://github.com/redhat-developer/vscode-microprofile/pull/185).
```

Comment:
I think we should remove these entries, since they aren't interesting to the end user.

Reply:
fixed